### ISSUE TITLE
Wrong Day Of Week

### ISSUE THREAD

**Issue by osheynin** [apptastic-software/trading-calendar#1]
Thursday instead of Tuesday is received with this request:
http://127.0.0.1:8000/api/v1/markets/status?mic=XSTO
```
{
"mic": "XSTO",
"day_of_week": "Thursday",
"is_weekend": false,
"is_business_day": true,
"status": "Closed",
"is_early_close": false,
"local_time": "2024-03-05T19:59:13+01:00",
"open_time": "2024-03-05T09:00:00+01:00",
"close_time": "2024-03-05T17:30:00+01:00"
}
```
Similar error is observed for XNIS, and other MICs

**Comment by osheynin** [comment]
I've found the bug. 
It is trivial typo here: https://github.com/apptastic-software/trading-calendar/blob/fb08c6a587b32856d9a725efc76fa2f28d0d8856/trading_calendar/main.py#L138
:)

**Comment by w3stling** [comment]
Good finding. Solved in PR #2 

It is now fixed in release 0.0.14
